Leanplum-SDK/Classes/Leanplum.m

```diff
@@ -3634,6 +3634,25 @@ - (void)runActionNamed:(NSString *)name
                 }];
             });
             return;
+        } else {
+            // Message doesn't seem to be on the device,
+            // so let's forceContentUpdate and retry showing it.
+            [Leanplum forceContentUpdate: ^(void) {                
+                LPActionContext *chainedActionContext =
+                        [Leanplum createActionContextForMessageId:messageId];
+                chainedActionContext.contextualValues = self.contextualValues;
```

Comment:
Make sure that message is there. Or else this will create NPE.

Reply:
Fixed.